src/components/location/dto/location.dto.ts

```diff
@@ -55,7 +55,7 @@ export class Country extends Resource implements Place {
   @Field(() => SecuredString)
   name: SecuredString;
 
-  @Field(() => SecuredRegion)
```

Comment:
I meant all of the changes in this file. Secured fields don't need their types explicitly set because they can be inferred from the property type.